src/index.html

```diff
@@ -22,6 +26,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="container_logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="container_menu">
+        <ul class="menu">
+          <li>
+            <a
+              class="is-active"
```

Comment:
The class 'is-active' should follow the BEM naming convention. If 'link' is the base element class, then 'link--active' would be a better name.